src/tests/tls-load/test/home-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo /etc/freeradius/lib/local/.libs >> /etc/ld.so.conf
```

Comment:
This isn't necessary.  If you're running the server out of the local source tree, you can wrap the execution of the scripts with `jlibtool`.  See `scripts/run/radiusd`.  You can just do

```
/path/to/source/scripts/run/radiusd ....
```

and it will pass in the correct command-line arguments too.

The `jlibtool` wrapper sets up a bunch of environment variables which allow things to be run correctly.

I'm also not sure why the server binary and libraries are in `/etc`?  That directory is for configuration.  It may be better to use a different directory, like `/build`, or even `/home/freeradius`.

This just makes it clearer where everything is.

Reply:
Alright, fixed.